src/commands/BeautifyCommand.ts

```diff
@@ -143,6 +141,29 @@ export class BeautifyCommand extends BaseCommand {
       });
     });
   }
+
+  private validateLanguage(
+    language: string | undefined,
+    unibeautify: Unibeautify
+  ) {
+    if (!language) {
+      const error = new Error("A language is required.");
+      return this.handleError(error, 1);
+    }
+    const langs = unibeautify.findLanguages({ name: language });
+    if (langs.length === 0) {
+      const bestMatchLanguage = getAllLanguages().find(lang => {
+        return lang.toLowerCase() === language.toLowerCase();
```

Comment:
I would be OK if you also wanted to add https://www.npmjs.com/package/fast-levenshtein

cc @lassik @muuvmuuv

Something like:

```typescript
const languages = getAllLanguages();

const distances: number[] = languages.map(lang =>
  levenshtein.get(lang.toLowerCase(), language.toLowerCase())
);
const bestDistance: number = Math.min(...distances);

const bestMatchLanguages = getAllLanguages().filter((lang, index) =>
  distances[index] === bestDistance
);

// ...

const error = new Error(
  `Language '${language}' was not found. Did you mean '${bestMatchLanguages.join(" or ")}'?`
);
```

The above only gets the best. It could also be modified to support an arbitrary threshold:

```diff
- distances[index] === bestDistance
+ const distanceThreshold = 2;
+ distances[index] <= (bestDistance + distanceThreshold)
```

So `C` would also show `C++` and visa versa 😉 .